apps/emqx_schema_registry/src/emqx_schema_registry.app.src

```diff
@@ -13,7 +13,8 @@
         gpb,
         jesse,
         avlizer,
-        emqx
+        emqx,
+        emqx_bridge_http
```

Comment:
```suggestion
        emqx_resource
```

Reply:
`emqx_bridge_http` includes `emqx_resource`, and we do use modules from `emqx_bridge_http`.